_posts/2018-12-25-Building-A-Continues-Data-Processing-System.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title:  "Building a continues data processing system"
+date:   2018-12-25 23:00:00 +0800
+categories: [blog]
+tags: [elixir, backend, back-pressure]
+author: Jun Lin
+---
+
+## A brief introduction to Archiver
+
+At RingCentral, we have a system called **Archiver**, which is a system automatically uploads customer’s call recordings, SMS and fax to their cloud storages.
+
+For example, a company connect it’s Dropbox account with Archiver, then whenever it’s employees make phone calls and recording the audio, Archiver will upload the call logs to the customer’s Dropbox storage.
+
+It seems like a really simple system, but it is still challenging to *doing great*. The heavy lifting of Archiver is it’s background workers, as some of our customers are large companies like an airline company, they may generate many call recordings in a day.
```

Comment:
doing great is bad grammar really, it is still challenging to "do greatly", or "be doing it greatly". Again, sounds a bit weird in my opinion.
"it's background workers" => "its background workers"
"large companies like an airline company" => "large companies like airline companies"